scrapers/src/scrapers/co/sos/general_candidates.rs

```diff
@@ -231,6 +248,18 @@ impl Scraper {
             ..Default::default()
         })
     }
+
+    fn build_politician_input(entry: &CandidateEntry) -> db::UpsertPoliticianInput {
+        let slug = PoliticianSlugGenerator::new(SOURCE_ID, entry.name.as_str()).generate();
+        db::UpsertPoliticianInput {
+            slug: Some(slug),
+            full_name: Some(entry.name.clone()),
+            first_name: Some("".into()), // TODO
+            last_name: Some("".into()),  // TODO
```

Comment:
Can we add the party_id reference here that is upserted above?

Reply:
ah good catch. Meant to do that but probs got distracted by parsing politician names 🙃